### PR TITLE
Add more troubleshooting tips

### PR DESCRIPTION
**Pull request by vikaschoudhary16**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -264,3 +264,25 @@ kubectl logs -f -n aws-provider-system aws-provider-controller-manager-0
...
+If `aws-provider-controller-manager-0` logs did not help, you might want to look into cloud-init logs, /var/log/cloud-init-output.log, on the controller host.
+Also, verifying kubelet status and logs may also provide hints:
+```bash
+journalctl -u kubelet.service
+systemctl status kubelet
+```
+For reaching controller host:
+
+* Find out public IP of the bastion node. 
+* copy private key(or the .pem), corresponding to the key-pair discussed in the `ssh key pair` section above, to the bastion node:
```
Comment (randomvariable) [inline docs/getting-started.md]: Torn between saying to do this vs. using SSH agent forwarding with `ssh -A <node>`

Comment (vikaschoudhary16) [reply docs/getting-started.md]: @randomvariable updated. PTAL!

File: docs/getting-started.md [hunk 2]
```diff
@@ -264,3 +264,59 @@ kubectl logs -f -n aws-provider-system aws-provider-controller-manager-0
...
+                                "Service": "ec2.amazonaws.com"
+                            }
+                        }
+                    ]
+                }, 
+                "RoleId": "AROAT5TK4ZONE72CCEQRH", 
+                "CreateDate": "2019-05-13T16:45:12Z", 
+                "RoleName": "control-plane.cluster-api-provider-aws.sigs.k8s.io", 
+                "Path": "/", 
+                "Arn": "arn:aws:iam::269733383066:role/control-plane.cluster-api-provider-aws.sigs.k8s.io"
```
Comment (randomvariable) [inline docs/getting-started.md]: Replace all instances of the account id there with `123456789012`

File: docs/getting-started.md [hunk 3]
```diff
@@ -264,3 +264,59 @@ kubectl logs -f -n aws-provider-system aws-provider-controller-manager-0
...
+                        {
+                            "Action": "sts:AssumeRole", 
+                            "Effect": "Allow", 
+                            "Principal": {
+                                "Service": "ec2.amazonaws.com"
+                            }
+                        }
+                    ]
+                }, 
+                "RoleId": "AROAT5TK4ZONE72CCEQRH",
```
Comment (randomvariable) [inline docs/getting-started.md]: Use the example `AROAJQABLZS4A3QDU576Q` from https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html

File: docs/getting-started.md [hunk 4]
```diff
@@ -264,3 +264,59 @@ kubectl logs -f -n aws-provider-system aws-provider-controller-manager-0
...
+### kubelet on the control plane host failing with error: NoCredentialProviders
+```bash
+failed to run Kubelet: could not init cloud provider "aws": error finding instance i-0c276f2a1f1c617b2: "error listing AWS instances: \"NoCredentialProviders: no valid providers in chain. Deprecated.\\n\\tFor verbose messaging see aws.Config.CredentialsChainVerboseErrors\""
+```
+This error can occur if `CloudFormation` stack is not created properly and IAM instance profile is missing appropriate roles. Run following command to inspect IAM instance profile:
+```bash
+$ aws iam get-instance-profile --instance-profile-name control-plane.cluster-api-provider-aws.sigs.k8s.io --output json
+{
+    "InstanceProfile": {
+        "InstanceProfileId": "AIPAT5TK4ZONLXALXYCG2",
```
Comment (randomvariable) [inline docs/getting-started.md]: Use example ID `AIPAJQABLZS4A3QDU576Q` from https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html

**Comment by vikaschoudhary16** [comment]
@randomvariable updated, PTAL!

**Pull request merged**